<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Manage Departments 1"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="59.259">
  <properties />
            <testcase name="a_test" time="59.259" classname="hcm.tests.ManageDepartmentsTest">
                <failure type="java.util.regex.PatternSyntaxException"
                  message="Dangling meta character &apos;*&apos; near index 0
*
^"
              >
<![CDATA[java.util.regex.PatternSyntaxException: Dangling meta character '*' near index 0
*
^
  java.util.regex.Pattern.error(Pattern.java:1924)
  java.util.regex.Pattern.sequence(Pattern.java:2090)
  java.util.regex.Pattern.expr(Pattern.java:1964)
  java.util.regex.Pattern.compile(Pattern.java:1665)
  java.util.regex.Pattern.<init>(Pattern.java:1337)
  java.util.regex.Pattern.compile(Pattern.java:1022)
  java.lang.String.replaceAll(String.java:2162)
  hcm.tests.ManageDepartmentsTest.createDepartment(ManageDepartmentsTest.java:97)
  hcm.tests.ManageDepartmentsTest.a_test(ManageDepartmentsTest.java:29)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:606)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:773)
  org.testng.TestRunner.run(TestRunner.java:623)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
  org.testng.SuiteRunner.run(SuiteRunner.java:259)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
  org.testng.TestNG.run(TestNG.java:1018)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
